examples/with-tailwindcss-emotion/package.json

```diff
@@ -4,25 +4,22 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start",
-    "build:base-css": "tailwindcss build ./styles/tailwind.base.css -o ./styles/base.css"
+    "start": "next start"
   },
   "dependencies": {
-    "@emotion/css": "^11.0.0-next.12",
-    "@emotion/react": "^11.0.0-next.12",
-    "@emotion/styled": "^11.0.0-next.12",
-    "next": "latest",
-    "react": "16.13.1",
-    "react-dom": "16.13.1"
+    "@emotion/react": "^11.1.1",
+    "@emotion/styled": "11.0.0",
+    "react": "17.0.1",
+    "react-dom": "17.0.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.9.0",
-    "@emotion/babel-plugin": "^11.0.0-next.12",
-    "@emotion/server": "^11.0.0-next.12",
-    "@tailwindcss/ui": "^0.2.2",
-    "@tailwindcssinjs/macro": "^0.3.1",
-    "babel-plugin-macros": "2.8.0",
-    "tailwindcss": "^1.6.0"
+    "@babel/core": "^7.12.8",
+    "@emotion/babel-plugin": "11.0.0",
+    "autoprefixer": "10.0.2",
+    "next": "latest",
+    "postcss": "8.1.10",
+    "tailwindcss": "^2.0.1",
+    "xwind": "0.7.1"
```

Comment:
Looks like this package is not open source, or I can't seem to find its repo, or maybe its usage is too low. Let's remove it from the example 🙏

Reply:
xwind can be found at these places:
- npm: https://www.npmjs.com/package/xwind
- github: https://github.com/Arthie/xwind

The [`@tailwindcssinjs/macro`](https://www.npmjs.com/package/@tailwindcssinjs/macro) package has been renamed to `xwind`. The `@tailwindcssinjs/macro` package is deprecated. [More info](https://github.com/Arthie/xwind/issues/13)

It's not possible to transform Tailwind classes to be used with emotion without a package like `xwind`.

Thanks for the feedback!

Reply:
LGTM!